tests/package.json

```diff
@@ -20,12 +20,11 @@
     "uuidv4": "^6.2.11"
   },
   "devDependencies": {
-    "@holochain/client": "0.8.0",
-    "@holochain/tryorama": "0.7.0",
+    "@holochain/client": "^0.11.14",
```

Comment:
Dang it :p Sure, we should wind those back for now.

> I'm actually not certain about how strict we need to be with the client versions and how semver is being used in the client.

I would recommend specifying `^0.11.13` and committing the workspace package lockfile. That should allow `0.11.14` to be used once we're ready to update whilst also preventing breakages due to semi-randomly selected "most recent versions" (`package-lock.json` ensures that any devs will end up with identical packages rather than potentially varied ones).

Reply:
(still to do: @pospi to organize lockfile)

@weswalla if you can't get it to not download 0.11.14 let me know and I can try to fix. I've never tried it before but I suspect that an `npm i` on a clean repo will prefer the most recent compatible package from npm (.14) and I thought maybe hacking at `package-lock.json` after generation to override that to .13 followed by wiping node_modules and another `npm i` could get us what we want for this version.

Reply:
I think the issue is that there are breaking changes even when only the rightmost number changes, so if we don't pin an exact client version there may be imcompatibilities with imported modules that also depend on the client (like `@neighbourhoods/nh-we-applet`)

Reply:
so for example, when this is on `0.11.13` then initializing `SensemakerStore()` gives an error because `@neighbourhoods/nh-we-applet` is on `0.11.9`, which I'm pretty sure is not following semver...